class_exercise_2.rb

```diff
@@ -18,5 +18,36 @@ def even_numbers_fibonacci
   
 end
 
-fibo = Fibonacci.new(10)
-puts fibo.even_numbers_fibonacci
+#--------------TEST-------------------
+
+require 'minitest/autorun'
+
+class FibonacciTest < Minitest::Test
+
+  def setup
+    @number = Fibonacci.new(10)
+  end
+
+  def test_number_is_instance_of_Fibonacci
```

Comment:
Please call the method: `test_name_of_the_method` => `test_ even_numbers_fibonacci `